libs/ng-morph/ng/bootstrap/add-provider-to-bootstrap-application-fn.ts

```diff
@@ -0,0 +1,12 @@
+import { CallExpression } from "ts-morph";
+import { pushToObjectArgument } from "../helpers/push-to-object-argument";
+
+export function addProviderToBootstrapApplicationFn(
+  callExpression: CallExpression,
+  provider: string,
+  { unique = false }: { unique?: boolean } = {}
+) {
+  const [_, options = callExpression.addArgument(`{providers: []}`)] = callExpression.getArguments();
```

Comment:
btw in Taiga, we found that in some projects they write like this:

```
import {OPTIONS} from "./options"

bootstrapApplication(AppComponent, OPTIONS)
```

(second argument is not object, but constant)

we handled it like this:

```
const optionObject = getOptionObject(options)

function getOptionsObject(
    options: Identifier | ObjectLiteralExpression,
): ObjectLiteralExpression {
    if (Node.isObjectLiteralExpression(options)) {
        return options;
    }

    const definition = options.getDefinitionNodes()[0];

    return definition.getChildrenOfKind(SyntaxKind.ObjectLiteralExpression)[0];
}

...
```

Reply:
I think it's a good think for support, I'll add it, thanks!